src/components/draggable/draggable.stories.mdx

```diff
@@ -1,5 +1,6 @@
 import { Meta, Story, Canvas } from "@storybook/addon-docs";
 import { ArgsTable } from "@storybook/components";
+import * as stories from "./draggable.stories.tsx";
 
 import StyledSystemProps from "../../../.storybook/utils/styled-system-props";
 import { DraggableContainer, DraggableItem } from ".";
```

Comment:
Think you can remove the imports for DraggableItem and Checkbox here, as only DraggableContainer is needed

Reply:
I think it is needed, the prop tables little a little weird to me we can remove the ArgsTable and go with the below and it should display the padding props and the `id` and `children` ones etc

```
<StyledSystemProps of={DraggableItem} noHeader padding defaults={{ py: "8px" }} />
```